_includes/item_with_img.html

```diff
@@ -26,7 +26,7 @@ <h3 class="text-center-sm">
     </h3>
     {{ item.content }}
     {% if item.quick-install %}
-    <a href="#" data-toggle="popover" title="{{ item.title }} Quick Install" data-target="#{{ item.title | slugify }}-qi">[Quick Install]</a>
+    <a tabindex="0" role="button" data-toggle="popover" title="{{ item.title }} Quick Install" data-target="#{{ item.title | slugify }}-qi">[Quick Install]</a>
```

Comment:
Mhh, turns out `a` tags do not always need an `href`:
http://getbootstrap.com/javascript/#dismiss-on-next-click